### ISSUE TITLE
Test infrastructure running multiple ganache instances impacted with removal of server options

### ISSUE THREAD

**Issue by nepoche** [ConsenSys-archive/ganache#2040]
I am testing a bridge application which tries to spawn multiple ganache-server instances for setup. I have the following function which was working fairly well on `ganache-cli: "^6.12.2"`:

```
import ganache from 'ganache-cli';

export type GanacheAccounts = {
  balance: string;
  secretKey: string;
};

export function startGanacheServer(
  port: number,
  networkId: number,
  populatedAccounts: GanacheAccounts[],
  options: any = {}
) {
  const ganacheServer = ganache.server({
    accounts: populatedAccounts,
    port: port,
    network_id: networkId,
    _chainId: networkId,
    chainId: networkId,
    _chainIdRpc: networkId,
    ...options,
  });

  ganacheServer.listen(port);
  console.log(`Ganache Started on http://127.0.0.1:${port} ..`);

  return ganacheServer;
}
```
I am testing another process in parallel - a relayer - which listens to events on one ganache instance and issues transactions on the other ganache instances. I am afraid that the latest `"ganache": "^7.0.0-beta.2"` will not support this behavior.

**Comment by davidmurdoch** [comment]
What particularly isn't working for you? Are you available to hop on a zoom call with us real quick to discuss?

**Comment by nepoche** [comment]
Yes, I'd be happy to get in a call and discuss. In particular, it seems multiple ganache instances cannot be instantiated in the same manner.

**Comment by davidmurdoch** [comment]
Will you be available at 12:00 PM Eastern Time tomorrow (Jan 13)?

**Comment by davidmurdoch** [comment]
@nepoche Cool. We'll see you then at https://consensys.zoom.us/j/93659581189 !

**Comment by davidmurdoch** [comment]
@nepoche you joined an hour early. Does 12 Eastern still work for you?

**Comment by nepoche** [comment]
Yeah, just waiting and working on other things in the meantime.

**Comment by nepoche** [comment]
@davidmurdoch It seems the latest `"ganache": "^7.0.0-rc.0"` works without changes from the `ganache-cli: "^6.12.2"`. There are still issues with attempting to parameterize the server with blockTime and multiple instances; however, it seems this was an issue in 6.12.2 to begin with. I'd be happy to close this issue and open a different one to track the blockTime configuration.

**Comment by davidmurdoch** [comment]
Hm, a `Ganache.server({blockTime: 5})` mines a new block every 5 seconds for me. What are you seeing (feel free to open a new issue for this one)?

**Comment by nepoche** [comment]
I'll close this and try to create a minimal reproduction before opening another issue. Thanks again for the earlier meeting!
